### DISCUSSION TITLE
How do I hide the find in file search bar

### DISCUSSION THREAD

**Discussion by migui3230** [vscode-neovim/vscode-neovim#1540] [Q&A] [answered]
When i use cmd+f to find text in a file, i cant seem to exit out of it with escape, it just puts me back to normal mode in neovim. 

Has anyone worked a way around this?

**Comment by xiyaowong** [comment]
Works fine for me. Consider providing a screen recording.

**Comment by migui3230** [comment]
@xiyaowong 

video:
https://github.com/vscode-neovim/vscode-neovim/assets/74937076/dda66821-0998-48fb-a261-08741d6459f3

vscode keyboard shortcuts json file
![CleanShot 2023-10-16 at 23 06 48@2x](https://github.com/vscode-neovim/vscode-neovim/assets/74937076/4dcb91ae-d2ac-4843-9520-ca9df0dcf202)

**Reply by xiyaowong** [reply]
![image](https://github.com/vscode-neovim/vscode-neovim/assets/47070852/f724b631-ab53-4352-81f3-4a4213929532)
This binding is of the highest priority. You need to set the when expressions for constraint, but please do not add this binding. This is unnecessary.

**Reply by migui3230** [reply]
which when statements would be best for this command?

if i remove this command i can't go back to normal mode with the escape key

**Reply by xiyaowong** [reply, marked as answer]
![img](https://github.com/vscode-neovim/vscode-neovim/assets/113702109/1942a4d9-5da6-4c3a-a2b0-b8d5078d3ff3)
